lib/cli/writer.js

```diff
@@ -23,7 +23,7 @@ const os = require('os');
 const unmount = require('./unmount');
 const imageStream = require('../image-stream');
 const errors = require('../shared/errors');
-const constraints = require('../shared/constraints');
+const constraints = require('../shared/drive-constraints');
```

Comment:
We have plans to build such a testing infrastructure soon, and even got the green sign to order any equipment we need. We just need to put some time into it :)